medical_emr/README.rst

```diff
@@ -0,0 +1,188 @@
+[![Build Status](https://travis-ci.org/OCA/vertical-medical.svg?branch=8.0)](https://travis-ci.org/OCA/vertical-medical)
+[![Coverage Status](https://coveralls.io/repos/OCA/vertical-medical/badge.png?branch=8.0)](https://coveralls.io/r/OCA/vertical-medical?branch=8.0)
+
+What is Odoo Medical
```

Comment:
The README.rst doesn't follow the standard structure defined by OCA https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst